library/traefik

```diff
@@ -1,15 +1,15 @@
 Maintainers: Emile Vauge <emile@vauge.com> (@emilevauge), Vincent Demeester <vincent@sbr.pm> (@vdemeester), Ludovic Fernandez <ludovic@containo.us> (@ldez), Julien Salleyron <julien@containo.us> (@juliens), Nicolas Mengin <nicolas@containo.us> (@nmengin), Michael Matur <michael@containo.us> (@mmatur), Damien Duportal <damien@containo.us> (@dduportal)
 GitRepo: https://github.com/containous/traefik-library-image.git
 
-Tags: v2.0.1-windowsservercore-1809, 2.0.1-windowsservercore-1809, v2.0-windowsservercore-1809, 2.0-windowsservercore-1809, montdor-windowsservercore-1809, windowsservercore-1809
+Tags: v2.0.2-windowsservercore-1809, 2.0.2-windowsservercore-1809, v2.0-windowsservercore-1809, 2.0-windowsservercore-1809, montdor-windowsservercore-1809, windowsservercore-1809
 Architectures: windows-amd64
-GitCommit: e158b458340bad5e6c039ba84a4768e0d7e2195f
+GitCommit: a232b9ebed96febf723daa43c6899ec86ba554eb
 Directory: windows/1809
 Constraints: windowsservercore-1809
 
-Tags: v2.0.1, 2.0.1, v2.0, 2.0, montdor, latest
+Tags: v2.0.2, 2.0.2, v2.0, 2.0, montdor, latest
 Architectures: amd64, arm32v6, arm64v8
-GitCommit: e158b458340bad5e6c039ba84a4768e0d7e2195f
+GitCommit: a232b9ebed96febf723daa43c6899ec86ba554eb
```

Comment:
These should both be `3d06252965f0028af9de0787b8caa4a1623a9e96`, right? (`a232b9ebed96febf723daa43c6899ec86ba554eb` doesn't seem to exist on any branches in https://github.com/containous/traefik-library-image)

Reply:
we had a little problem with the publication of the release binaries, I had to make a push force and redo the release of the version.

Reply:
The problem is fixed now.